### DISCUSSION TITLE
10.0.0-BETA in update channel!?!?!

### DISCUSSION THREAD

**Discussion by justbane** [pimcore/pimcore#8731] [General]
Should 10.0.0-BETA1 be in the official update channel?

My production sites are showing my next update is 10.0.0-BETA1. I think that should not be in the official update channel.

Am I correct in this assumption?

**Comment by brusch** [comment]
@justbane thanks for the hint -> done 😊

**Reply by justbane** [reply]
no worries, thanks @brusch !
